src/main/java/org/primefaces/component/splitter/SplitterRenderer.java

```diff
@@ -122,7 +122,7 @@ protected void encodeScript(FacesContext context, Splitter splitter) throws IOEx
                 .attr("stateKey", splitter.getStateKey())
                 .attr("stateStorage", splitter.getStateStorage());
 
-        wb.callback("onResizeEnd", "function(event,ui)", splitter.getOnResizeEnd());
+        wb.callback("resizeEnd", "function(event,ui)", splitter.getOnResizeEnd());
```

Comment:
I think this is still supposed to be `onResizeEnd` since this is the client side JS callback.

So client side would be `<p:splitter onresizeend="alert('test');" />` but the AJAX is `event="resizeEnd`.  Also I think the Client Side events are usually all lowercase like `onresizeend`

Reply:
I did that because the event name is "onResizeEnd" in primeng or primereact. And I think also "resizeEnd" :D If you think okay, I will merge

Reply:
I think in most PF components the AJAX is always without "on" and the client side events always have "on" so we should keep that consistent.

Reply:
I agree. I wasn't sure about the naming anyway. In the end, I could not be sure which one should be what. And I did it all the same. To fix it when you notice

Reply:
We reviewed the codes with Mert and decided to do "onResizeEnd". The reason is that this is not html event. We have used camel cases in many places. accordion panel etc.

Reply:
OK